charmcraft/providers.py

```diff
@@ -112,26 +106,29 @@ def ensure_provider_is_available() -> None:
 
     :raises CommandError: if provider is not available.
     """
-    if is_provider_available():
-        return
-
-    if confirm_with_user(
-        "LXD is required, but not installed. Do you wish to install LXD "
-        "and configure it with the defaults?",
-        default=False,
-    ):
-        try:
-            lxd_installer.install()
-        except LXDInstallationError as error:
+    if not is_provider_available():
+        if confirm_with_user(
+            "LXD is required, but not installed. Do you wish to install LXD "
+            "and configure it with the defaults?",
+            default=False,
+        ):
+            try:
+                lxd.install()
+            except lxd.LXDInstallationError as error:
+                raise CommandError(
+                    "Failed to install LXD. Please visit https://snapcraft.io/lxd for "
```

Comment:
Remove Please, start with Visit -> https://discourse.ubuntu.com/t/tone-of-voice/18883

Reply:
Ah yes, I questioned it when I wrote it way back when and never did reconsider. :thinking:  I think I just feel compelled to be polite :rofl:  will fix (but note that this was just an indent change, not new text).

Reply:
done!